README.md

```diff
@@ -0,0 +1,140 @@
+* [Advanced topics](#advanced-topics)
+* [Support](#support)
+* [License](#license)
+
+
+## Introduction
+
+This is the code repository for **FIWARE Testbed Deploy component**. This project is a scripts sets developed to deploy Openstack testbeds
+
+This project is part of [FIWARE](http://www.fiware.org).
+
+Any feedback on this documentation is highly welcome, including bugs, typos
+or things you think should be included but aren't. You can use [github issues](https://github.com/telefonicaid/fiware-testbed-deploy/issues/new) to provide feedback.
+
+[Top](#top)
+
+## Overall description
+A set of scripts to create a secure environment for testing is provided. The
+environment is created in a virtual machine using the FIWARE Lab infrastructure.
+The environment variables OS_AUTH_URL, OS_REGION_NAME, OS_USERNAME, OS_PASSWORD
+and OS_TENANT_NAME must be filled with the data of a valid FIWARE Lab account.
+
+The installation of the testbed is fully automatized and consists on an OpenStack
+instance where the original keystone server was replaced with the KeyRock server.
+This is work in progress; the current version only installs Glance, Nova and Neutron,
+but skuld also purges Swift, Cinder and Blueprint resources.
+
+### Deploying one testbed
+By using a fiware-testbed-deploy script (*launch_vm.py*), it is possible to deploy an entire Openstack testbed.
+The script uses a floating IP and creates both a keypair (it saves the SSH private key as
+~/.ssh/testbedskuld_key) and a security group. The security group opens
+the port 22 (SSH) and allows the ICMP traffic. It is also possible to connect
+to any port from a VM using the same security group.
+
+The names of the keypair, the security group and other parameters as the preferable
+floating IP are configurable in the *settings.py* file, but most of the users may
+ignore this file securely. However, to use the script in a different platform
+than FIWARE Lab, probably is necessary to change the parameter about the
+public shared network name.
+
+Although it is already possible to connect to the ubuntu account of the server, the installation is still running
+inside the VM and needs a few minutes to complete. Usually the installation
+process lasts between 10 and 20 minutes. The job is finished after the file
+*config_vars* is copied into the */home/ubuntu* folder inside the virtual machine.
+
+When the installation is finished, the credential may be loaded with *. ~/config_vars*.
+The command *nova list* shows a testing VM that has been created during the installation
+inside the testbed (that is, it is a virtual machine running inside the testbed
+virtual machine). The floating IP 192.168.58.201 is associated to this
+VM (that is the second IP of the pool, the first was assigned to the router). It is
+possible to connect to the server following this steps:
+
+    eval $(ssh-agent -s)
+    ssh-add ~/.ssh/testkey
+    ip a add dev br-ex 192.168.58.1/24
+    ssh cirros@192.168.58.201
```

Comment:
Add a note: Please remember that in this case we are using a CentOS image, if you use a debian or ubuntu the user have to be different.